### ISSUE TITLE
influx 2.5 which is in the latest docker compose not working with existing Grafana dashboards

### ISSUE THREAD

**Issue by fate8383** [unpoller/unpoller#466]
I have been looking all the places today but I cannot figure it out, Unpoller working, writing to influx 2.5, influx is added as FLUX data source to Gafana, saves and tests all checks. But all Grafana dashboard has no info to display. Am I missing something? thanks

**Comment by platinummonkey** [comment]
nope you're not, the dashboards need some TLC for influx 2.x support https://github.com/unpoller/dashboards/tree/master/v2.0.0

**Comment by fate8383** [comment]
there it is :) I think this is the only place I didn't look. Thanks a lot. I honestly rolled back to 1.8 my influx like 5 minutes ago after failing 2-3 hours. I will try again. we add as normal as FLUX now right, not QL? we don't need any DRMP to v1?

**Comment by platinummonkey** [comment]
> we add as normal as FLUX now right, not QL? 

Yeah I think that's the preferred default for influxdb 2.x, I'm not sure if it still supports the old syntax or not

> we don't need any DRMP to v1?

Sorry, I have no idea what this means 😅 I don't use grafana and influx myself. I did leave backwards compatibility to previous influx users.

**Comment by fate8383** [comment]
My bad also wrote it incorrectly, it is like to map v1x database to v2x bucket so you can still use the QL, but I will try the FLUX which works already I know but was struggling with dashboards and now I know why :) thanks a lot for this

**Comment by fate8383** [comment]
so all the config is there and I am trying but not getting anything in grafana, I use raw data used as json but nothing is grabbed. I can see all in influx UI but not in Grafana?

**Comment by fate8383** [comment]
Oh well I mapped all with old database and using v1 style in v2 and in grafana they are working again, not the best way but it works. I just cannot get clients graph work no matter what I do... there seems to be a bug I am not very sure but just one graph is not working out of all makes me thing that. Not the dpi one but normal client insights graph.

**Comment by fate8383** [comment]
finally I figured it out, clients dash is basically only works with v1.6.3 json file in the dashboard git page, none of the v2 dashboards or grafana IDs are working, but json from v1 is working perfectly fine with my mapped v1 database to influx2 latest version.

**Comment by Xboarder56** [comment]
Just spent some time going through the guide with the docker-compose file. Is there plans to support InfluxDB2 (which is the default for compose) in the dashboards? It seems odd to upgrade the directions for influxDB2 but have the dashboards unsupported.

**Comment by ilbarone87** [comment]
@Xboarder56 I believe it's unlikely unless someone updates all the queries in the grafana dashboard to be complaint with influxDB 2.x language.

**Comment by srgvg** [comment]
Brand new to this, I skimmed the docs, and noticed the warnings about not using InfluxDb `2.0` or older.
The compose file mentioning `2.5` however got me very confused, which lead me to here.  Not sure why that is kept there, isn't that a bug in the docs/compose file?

**Comment by davidnewhall** [comment]
The docker compose has been fixed to reflect version 1.8 as preferred. While 2.5 works, there are no dashboards for it, so it's not recommended.
